src/scss/components/_editgrid.scss

```diff
@@ -32,6 +32,16 @@
     }
   }
 
+  @include bem.element('summary') {
+    display: flex;
+    flex-direction: column;
+    gap: var(--of-editgrid-summary-gap, var(--of-editgrid-item-gap));
+
+    p {
+      margin: 0;
+    }
+  }
```

Comment:
have you considered using https://nl-design-system.github.io/utrecht/storybook/?path=/docs/css_css-data-list--docs?